project/MiMa.scala

```diff
@@ -925,7 +925,11 @@ object MiMa extends AutoPlugin {
         ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.mapError"),
 
         // #21894 Programmatic configuration of the ActorSystem
-        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorSystemImpl.this")
+        ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorSystemImpl.this"),
+
+        // #22240 Re-implement javadsl testkit
+        ProblemFilters.exclude[MissingTypesProblem]("akka.testkit.TestProbe"),
+        ProblemFilters.exclude[MissingTypesProblem]("akka.remote.testkit.MultiNodeSpec")
```

Comment:
`MultiNodeSpec` extends `Testkit` and the type hierarchy of `Testkit` is changed, so mima complained about it.